src/components/SignUpForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from "react-router-dom";
+
+import Form from "./Form";
+import Input from "./Input";
+import Button from "./Button";
+import SeparForm from "./SeparForm";
+import Linked from "./Linked";
+
+const SignUpForm = () => {
```

Comment:
Sign up was not in priority ?

Reply:
No, we don't include it in our app for the moment.